ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/base/v1/base/clientlibs/editor/css/base.less

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright 2021 Adobe
```

Comment:
Please fix the copyright year. Also, this file should just have bare minimum CSS to make the component usable. Can we write a comment in this file for the same?